apps/web/components/apps/installation/ConfigureStepCard.tsx

```diff
@@ -28,18 +28,22 @@ export const ConfigureStepCard: FC<ConfigureStepCardProps> = ({
   categories,
   credentialId,
   eventType,
-  onSave,
   loading,
 }) => {
   const { t } = useLocale();
   const { getAppDataGetter, getAppDataSetter } = useAppsData();
-  const { shouldLockDisableProps } = useLockedFieldsManager(
+  const formMethods = useForm();
+  const { shouldLockDisableProps } = useLockedFieldsManager({
```

Comment:
@ThyMinimalDev @sean-brydon, do we need the useLockedFieldsManager in the app installation flow? I see that apps are locked as a whole for managed event types and not on the app level. This is testing with GTM. I think if we pass the right data through here we don't need to use this function and worry about form methods. The formMethds taht useLockedFieldsManager expects is for a whole event type.